jerry-core/parser/js/js-parser-expr.c

```diff
@@ -2608,8 +2608,11 @@ parser_pattern_process_assignment (parser_context_t *context_p, /**< context */
   {
     parser_parse_expression (context_p, PARSE_EXPR_NO_COMMA | PARSE_EXPR_LEFT_HAND_SIDE);
 
+
     if (!PARSER_IS_PUSH_LITERAL (context_p->last_cbc_opcode))
     {
+      uint16_t opcode_inc = 2;
```

Comment:
This could be moved below after the raise error and just before the `last_cbc_opcode` check.

Reply:
And.... Why `2`? Would it be possible to have a bit of comment here on this?